pkg/app/api/service/webservice/service.proto

```diff
@@ -401,3 +405,15 @@ message ListAPIKeysRequest {
 message ListAPIKeysResponse {
     repeated model.APIKey keys = 1;
 }
+
+message GetInsightDataRequest {
+    string application_id = 1 [(validate.rules).string.min_len = 1];
+    pipe.model.InsightMetricsKind metrics_kind = 2;
+    pipe.model.InsightStep insight_step = 3;
+    int64 first_date = 4;
+    int64 last_date = 5;
+}
```

Comment:
Can we reorder the fields by their stability/importance. I mean the `metrics_kind` must be required so it should at the top.

```
metrics_kind
step
range_from
range_to
application_id
```